### DISCUSSION TITLE
Remove "Code" from EoS

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#278] [Q&A] [answered]
Title sums it all up.

How do I remove code from Elementary.

I've thusfar been stymied.

Working on Horus.

Thanks

**Comment by doobes** [comment]
I've tried uninstalling through  the app center . It's goes through the motions but remains installed.

Just tried it again.

It toggles back and forth between Free and Open a couple of times and then the app center closes.

Borked....

**Comment by lenemter** [comment, marked as answer]
You can uninstall Code from Terminal:

```
sudo apt remove io.elementary.code
```

Also can you report the issue about uninstalling Code through AppCenter being broken at https://github.com/elementary/appcenter/issues please?

**Reply by doobes** [reply]
Done and done. 

Thank you

**Comment by jeremypw** [comment]
Alternatively you can set another app to be the default text editor through System Settings/Applications/Default. If there are features you would like to see in Code please raise them on the Issues page if you have not already done so.

**Reply by doobes** [reply]
I use notepadqq because it's very similar to NotePad++ I've used for years on my Windows boxes. Nothing wrong with Code, just not my thing.

Notepadqq is/was set as the default text editor.

cheers,

chris

**Reply by jeremypw** [reply]
Fair enough.
